packages/common/src/ceramic-api.ts

```diff
@@ -146,78 +181,29 @@ export interface AdminApi {
 /**
  * Describes Ceramic node API
  */
-export interface CeramicApi extends CeramicSigner {
-  // loggerProvider: LoggerProvider; // TODO uncomment once logger is available on http-client
+export interface CeramicApi {
+  readonly loggerProvider: LoggerProvider
 
   readonly index: IndexApi
 
   readonly admin: AdminApi
 
+  // readonly streamManipulator: StreamManipulator
+  //
+  // readonly signer: CeramicSigner
```

Comment:
I think this was meant to be deleted

Reply:
It was. I'm actually wondering if I can delete this whole file now. Might give that a shot.